src/components/world.js

```diff
@@ -91,9 +220,26 @@ export default class World extends Component {
       width: '100%',
     };
 
+    const { renderWidth, renderHeight, scale } = this.context;
+
+    let debugRenderTarget = false;
+
+    if (this.props.debug) {
+      debugRenderTarget = (
+        <canvas
+          key="debug-render-target"
+          style={{position: 'relative'}}
```

Comment:
So it sits on top of the other relatively positioned elements in its stacking context.